### PR TITLE
Add @all

### PR DESCRIPTION
**Pull request by luisbosch**
The string @all gets replaced by all eku-members present in immedia.

### CODE REVIEW

**Comment by adamantivm** [comment]
whoa!

**Comment by adamantivm** [comment]
VERY polemic feature

**Comment by luisbosch** [comment]
Ok, we'll see how it goes. We can remove it or limit it's usage in the future.

File: app/js/directives.js [hunk 1]
```diff
@@ -1849,6 +1849,17 @@ angular.module('myApp.directives', ['myApp.filters'])
 
       function onMessageSubmit (e) {
         $timeout(function () {
+          // EKUCODE: Replace @all with all immedia participants
```
Comment (german-e-mas) [inline app/js/directives.js]: Loved the `Ekucode` tag btw

**Pull request merged**